app/controllers/api/v1/items/items_random_controller.rb

```diff
@@ -0,0 +1,8 @@
+class Api::V1::Items::ItemsRandomController < ApplicationController
+
+  def show
+    id = Item.pluck(:id).sample
+    @item = Item.find(id)
```

Comment:
This logic is fine, but does it belong in your controller? Where could we put this that's more appropriate? Perhaps a "random" method